packages/admin/cms-admin/src/dashboard/widgets/LatestBuildsDashboardWidget.tsx

```diff
@@ -15,40 +16,39 @@ type Row = {
 };
 
 export type LatestBuildsDashboardWidgetProps = {
-    tableQuery: Pick<TableQueryProps, "api" | "loading" | "error"> & {
-        tableData?: Partial<ITableProps<Row>> & Pick<ITableProps<Row>, "data" | "totalCount">;
-    };
+    data: Partial<DataGridProps<Row>>;
 };
 
-export const LatestBuildsDashboardWidget = ({ tableQuery: { tableData, api, loading, error } }: LatestBuildsDashboardWidgetProps) => {
+export const LatestBuildsDashboardWidget = ({ data: { rows = [], ...restDataProps } }: LatestBuildsDashboardWidgetProps) => {
     const intl = useIntl();
 
+    const columns: GridColDef<Row>[] = [
+        {
+            ...disableFieldOptions,
+            field: "runtime",
+            headerName: intl.formatMessage({ id: "dashboard.latestBuildsWidget.runtime", defaultMessage: "Runtime" }),
+            flex: 1,
+            renderCell: ({ row }) => (
+                <BuildRuntime
+                    startTime={row.startTime ? parseISO(row.startTime) : undefined}
+                    completionTime={row.completionTime ? parseISO(row.completionTime) : undefined}
+                />
+            ),
+        },
+        {
+            ...disableFieldOptions,
+            field: "status",
+            headerName: intl.formatMessage({ id: "dashboard.latestBuildsWidget.status", defaultMessage: "Status" }),
+            width: 150,
+            renderCell: ({ row }) => <BuildStatus status={row.status}>{row.status}</BuildStatus>,
+        },
+    ];
+
     return (
         <DashboardWidgetRoot header={<FormattedMessage id="dashboard.latestBuildsWidget.title" defaultMessage="Latest Builds" />}>
-            <TableQuery api={api} loading={loading} error={error}>
-                {tableData && (
-                    <Table<Row>
-                        {...tableData}
-                        columns={[
-                            {
-                                name: "runtime",
-                                header: intl.formatMessage({ id: "dashboard.latestBuildsWidget.runtime", defaultMessage: "Runtime" }),
-                                render: (row) => (
-                                    <BuildRuntime
-                                        startTime={row.startTime ? parseISO(row.startTime) : undefined}
-                                        completionTime={row.completionTime ? parseISO(row.completionTime) : undefined}
-                                    />
-                                ),
-                            },
-                            {
-                                name: "status",
-                                header: intl.formatMessage({ id: "dashboard.latestBuildsWidget.status", defaultMessage: "Status" }),
-                                render: (row) => <BuildStatus status={row.status}>{row.status}</BuildStatus>,
-                            },
-                        ]}
-                    />
-                )}
-            </TableQuery>
+            <Box sx={{ height: 318 }}>
```

Comment:
318 seems pretty specific. Is this always appropriate?

Reply:
@thomasdax98 Yes, DataGrid requires a fixed height and that is the value needed to show exactly 5 rows and the header.

Reply:
Why can't you use the `autoHeight` prop? https://mui.com/x/react-data-grid/layout/#auto-height